docs/source/_json/breadcrumbs.json

```diff
@@ -0,0 +1,19 @@
+GET /plone/front-page/@components/breadcrumbs
+Accept: application/json
+
+HTTP 200 OK
+content-type: application/json
+
+[
+  {
+    "data": {
```

Comment:
@ebrehault @tisto Regarding wrapping "items" into "data", that really was just an ad-hoc decision during the sprint when I cobbled together that mock implementation. Someone from the frontend team requested a mock for breadcrumbs, and someone else needed one for navigation. The expected example responses they gave me varied quite a bit in terms of format, but not in semantics. So I tried to identify a common outer frame that they shared, and what you're seeing here is what we went with 😉 

I can't imagine anyone from the frontend team is attached in any way to this particular structure. I'd say eliminate `data`. As for making it consistent with the overall structure of collection-like resources: I'd say "yes, where it makes sense". So let's also have a top-level `items`, maybe later `items_total` if it serves any purpose. But I wouldn't implement actual batching, IMHO that would just make our lives unnecessarily hard (we also didn't do that for relations).

JSON-LDfication? Maybe `id` could be promoted to `@id` and contain the fully qualified URL of the endpoint, which would still carry the same information. I simply don't know what `id` is being used for right now on the frontend, if at all.

So I would probably eliminate `data`, but I don't feel strongly about any of the other points.